sdk/messaging/azservicebus/admin/admin_client.go

```diff
@@ -38,7 +43,13 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 
 // NewClient creates a Client authenticating using a TokenCredential.
 func NewClient(fullyQualifiedNamespace string, tokenCredential azcore.TokenCredential, options *ClientOptions) (*Client, error) {
-	em, err := atom.NewEntityManager(fullyQualifiedNamespace, tokenCredential, internal.Version)
+	var retryOptions utils.RetryOptions
+
+	if options != nil && options.RetryOptions != nil {
+		retryOptions = *options.RetryOptions
+	}
+
+	em, err := atom.NewEntityManager(fullyQualifiedNamespace, tokenCredential, internal.Version, retryOptions)
```

Comment:
@jhendrixMSFT - just to note, this is a really temporary situation. The ATOM client is still, mostly, just using the old code from the original SB package. I want to move it over to look more like the generated clients so I can just use the same policy_retry, etc...